README.md

```diff
@@ -1182,6 +1182,33 @@ The following configuration properties are global for all row caches.
 
 </table>
 
+### Authentication and Authorization Configuration
+
+Authentication and authorization may be independently configured.  By default they are set to `AllowAllAuthenticator` and `AllowAllAuthorizer` respectively.  This is necessary to allow initial access to a new cluster.  After initial installation these values may be changed by changing the appropriate environment variables, `CASSANDRA_AUTHENTICATOR` and `CASSANDRA_AUTHORIZER`.
+Further information regarding how to change authentication and authorization of a Cassaandra cluster is available [here](https://docs.datastax.com/en/cassandra/2.1/cassandra/security/security_config_native_authenticate_t.html).  The necessary changes to the cassandra.yml file referenced there are accomplished by changing the environment variables already mentioned.
```

Comment:
Provide links for Cassandra 3.x:
https://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/secureConfigInternalAuth.html
https://docs.datastax.com/en/cassandra/3.0/cassandra/configuration/secureConfigNativeAuth.html